modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/AbstractAssetManager.java

```diff
@@ -154,13 +155,22 @@ public Snapshot takeSnapshot(MediaPackage mediaPackage) {
     // try to fetch the asset
     for (final AssetDtos.Medium asset : getDb().getAsset(RuntimeTypes.convert(version), mpId, mpeId)) {
       for (final InputStream assetStream : getLocalAssetStore().get(StoragePath.mk(asset.getOrganizationId(), mpId, version, mpeId))) {
+
+        Checksum checksum = null;
+        try {
+          checksum = Checksum.fromString(asset.getAssetDto().getChecksum());
+        } catch (NoSuchAlgorithmException e) {
+          logger.warn("Invalid checksum for asset {} of media package {}", e, mpeId, mpId);
```

Comment:
```suggestion
          logger.warn("Invalid checksum for asset {} of media package {}", mpeId, mpId, e);
```

…otherwise `e` would be put into the first placeholder.

Reply:
Thanks!